src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -4667,6 +4688,7 @@ private bool PerformCopyFileToRemoteSession(FileInfo file, string destinationPat
 
             bool targetSupportsAlternateStreams = RemoteTargetSupportsAlternateStreams(ps, remoteFilePath);
 
+#if !UNIX
```

Comment:
I think that code cleanup should be part of a different PR.  Created https://github.com/PowerShell/PowerShell/issues/4601